core/controllers/access_validators.py

```diff
@@ -171,7 +171,9 @@ class BlogHomePageAccessValidationHandler(base.BaseHandler):
         'GET': {}
     }
 
-    @acl_decorators.can_access_blog_dashboard
+    # Using type ignore[misc] here because untyped decorator makes function
+    # "get" also untyped.
+    @acl_decorators.open_access # type: ignore[misc]
```

Comment:
@sahiljoster32 Just checking whether you're ok with this? (Ditto in multiple places below.)

Reply:
Done